packages/data/src/world/oot/overworld.yml

```diff
@@ -1957,8 +1957,8 @@
 "Zora Domain Shortcut":
   region: ZORA_DOMAIN
   exits:
-    "Zora Domain Underwater": "true"
-    "Lake Hylia": "true"
+    "Zora Domain Underwater": "is_adult || can_swim"
+    "Lake Hylia Near Shortcut": "can_swim"
```

Comment:
You enter the loading zone before voiding when you hookshot to it right? This can just be true still, since coming from Domain or Underwater will guarantee a way through, and this currently would force bronze scale even if you have Iron Boots.